x-pack/plugins/security_solution/public/common/components/exceptions/builder/index.tsx

```diff
@@ -377,7 +377,7 @@ export const ExceptionBuilder = ({
           )}
           <EuiFlexItem grow={1}>
             <BuilderButtonOptions
-              isOrDisabled={disableOr}
+              isOrDisabled={isOrDisabled ? isOrDisabled : disableOr}
```

Comment:
If builder consumer specifies that `OR` should be disabled no matter what, makes sure it does so and is not overwritten by `disableOr` logic.